docs/datasource_api_userguide.md

```diff
@@ -324,7 +318,7 @@ delay-clean-table-lock: 60000
 split-table: true
 ```
 
-If your TiDB's version is earlier than 3.0, set `spark.tispark.write.without_lock_table` to `true` to enable write, but ACID is **not** guaranteed.
+If your TiDB's version is earlier than 4.0, set `spark.tispark.write.without_lock_table` to `true` to enable write, but ACID is **not** guaranteed.
```

Comment:
Do we need to fill more info here? @birdstorm

Reply:
We may add the detailed info when we throw exception in batch write?

Reply:
We may add the detailed info when we throw exception in batch write?